server/views/pages/adjustments/tagged-bail/review.njk

```diff
@@ -20,6 +21,21 @@
 
 {% block content %}
 
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      {% if model.showUnusedMessage %}
```

Comment:
This page already has an unused message. Should that be removed or is there meant to be a banner AND the original content?

Reply:
No, the original content should've been removed. I have taken this out 👍